modules/gdscript/gd_parser.cpp

```diff
@@ -1123,13 +1142,13 @@ GDParser::Node* GDParser::_parse_expression(Node *p_parent,bool p_static,bool p_
 				case OperatorNode::OP_EQUAL: priority=8; break;
 				case OperatorNode::OP_NOT_EQUAL: priority=8; break;
 
-				
+
 				case OperatorNode::OP_IN: priority=10; break;
-				
+
 				case OperatorNode::OP_NOT: priority=11; unary=true; break;
 				case OperatorNode::OP_AND: priority=12; break;
 				case OperatorNode::OP_OR: priority=13; break;
-				
```

Comment:
All of those changes shouldn't have happened, since they are unrelated, but let's say it might pass....